src/parsers/PropertiesParser.js

```diff
@@ -76,7 +76,7 @@ const nullLogger = {
 /**
  * @class Parser for Java style properties files as used with react-intl.
  */
-class PropertiesParser extends Parser {
+export class PropertiesParser extends Parser {
```

Comment:
JSDocs doesn't like it if you export and declare a class in the same line. So, in order to get it to show up properly, we have to do:

```javascript
class PropertiesParser extends Parser { ... }
export PropertiesParser;
```

Reply:
That's actually a syntax error, instead it would have to be
```js
export { PropertiesParser };
```
Either way, the named export is not really needed here so I just reverted it in 99038cf